app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,33 @@
+class CommentsController < ApplicationController
+  def new
+    @comment = Comment.new
+    # @current_user = current_user
+    # @comment.author = @current_user
+    # @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+  end
```

Comment:
- [ ] Kindly remove all commented codes in this file. Removing commented codes makes your work look clean and readable. 👍 🖥️